app/presenters/organization_presenter.rb

```diff
@@ -0,0 +1,9 @@
+class OrganizationPresenter < Burgundy::Item
+  def members
+    UserPresenter.wrap item.members.sort_by(&:last_name)
+  end
+
+  def developments
```

Comment:
Agreed. I will overwrite the method. I worked a little with that particular scope, including the more verbose `-> { select "DISTINCT ON (developments.id)" }` with limited luck. The latter works but breaks some Active Record methods (like #count)!